napalm_logs/config/iosxr.yml

```diff
@@ -11,7 +11,7 @@ prefix:
   line: '{messageId}: {host}:{date} {time}: {processName}[{processId}]: %{error}'
 
 messages:
-  ROUTING-BGP-5-MAXPFX:
+  - error: ROUTING-BGP-5-MAXPFX
```

Comment:
I have added `error` as the vendor agnostic name, and `tag` as the vendor specific name.